src/ddscxx/src/org/eclipse/cyclonedds/topic/qos/TopicQosDelegate.cpp

```diff
@@ -128,6 +128,13 @@ TopicQosDelegate::policy(const dds::core::policy::Ownership& ownership)
     ownership_ = ownership;
 }
 
+void
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro. (BTW: I see that in this particular case the embedding OMG_DDS_OWNERSHIP_SUPPORT macro was also missing for the "ownership" policy).

Reply:
Will do.